packages/sources/tp-icap/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@chainlink/tp-icap-adapter",
+  "version": "0.0.1",
+  "description": "Chainlink TP ICAP adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "tp-icap"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.26.1",
```

Comment:
This should be updated, there's been a lot of fixes and improvements since this version. I'll leave a few comments for some of the breaking-ish changes